TGC Comprehensive Rules.md

```diff
@@ -0,0 +1,183 @@
+        1.2.4.2: Every card in your deck must be legal, not banned and not be misprint. <br>
+
+1.3: A coin toss is done to decide who goes first. <br>
+    1.3.1: The coin toss decides who goes first, not who decides to go first. <br>
+    1.3.x: This means a player is decided to be heads and the other tails; whichever side the coin lands on goes first. <br>
+
+1.4: Each player draws 5 cards from their deck. <br>
+1.5: Then, each player may mulligan cards in their hand. <br>
+    1.5.1 During the mulligan each player may discard a card (face-down) from their hand to draw 1. <br>
+        1.5.1.x: The opponent is not allowed to see the discarded cards but can know how many cards you chose to discard.  <br>
+    1.5.2: They can perform this up to 3 times. <br>
+    1.5.3: After both players have finished mulliganing shuffle all discarded cards back into the deck. <br>
+
+1.6: Both players starts with 20 Heathshards (or HS) and 1 Plasma. <br>
+1.7: The game begins on the Draw Phase of the player who won the coin toss. <br>
+1.8: The player that plays second keeps the coin and places it besides them, flipped on heads. <br>
+     1.9.x: This is relevant for the \"Flip Coin\" action, detailed in 3.6 <br>
+
+1.9: Turns in this game are divided in 5 parts, called the phases. <br>
+    1.9.a: Draw Phase <br>
+    1.9.b: Standby Phase <br>
+    1.9.c: Play Phase <br>
+    1.9.d: Battle Phase <br>
+    1.9.e: End Phase <br>
+        1.9.x: There is no \"Play Phase 2\". You do not get to return to a Play Phase after the Battle Phase. <br>
+
+1.10: During the Draw Phase a number of automatic game actions happen, in the following order. <br>
+    1.10.1: The turn player\'s mana gets set to the amount of Draw Phases they have had in total, including this one. <br>
+        1.10.1.x: So on their first turn they get 1 mana, on their second 2 mana, on their third 3 mana, etc. <br>
+    1.10.2: The turn player draws a card. <br>
+    1.10.3: The turn player untaps all their tapped cards. <br>
+    1.10.4: The "Pass Phase" action is immediately performed. <br>
+    1.10.5: This Pass Phase does not create an Open gamestate. <br>
+
+1.11: Terminology definitions: <br>
+    1.11.a: When you draw a card you add the top card of your deck to your hand. <br>
+    1.11.b: When a card is destroyed it is moved to the discard pile <br>
+        1.11.b.1: When a card is destroyed all cards attached to it also get destroyed. <br>
+    1.11.c: When a card is tapped it is flipped sideways. <br>
+    1.11.d: When a card is untapped it is returned to the upright position <br>
+
+1.12: Any player that have their HP reduced to, set to or at 0 loses the game automatically. <br>
+1.12.1: When this happens the other player immediately wins the game. <br>
+    1.12.x: This can happen even if the game is in a Closed Gamestate <br>
+
+<h2> Open and Closed States </h2>  <br>
```

Comment:
I think this section is a bit overexplained, the point youre trying to get across is that players should not do anything while there is a choice being made (including drawing cards) which seems simpler than generating a gamestate for that.